op-chain-ops/crossdomain/legacy_withdrawal.go

```diff
@@ -170,3 +170,16 @@ func (w *LegacyWithdrawal) Value() (*big.Int, error) {
 
 	return value, nil
 }
+
+// CrossDomainMessage turns the LegacyWithdrawal into
+// a CrossDomainMessage
```

Comment:
Maybe worth a comment about it having a zero gaslimit? Is that even valid?

Reply:
I've added a comment here, basically there is no concept of a gas limit with the legacy withdrawals, its also technically a "min gas limit" because the gas limit set on the tx overrides